tests/integration/integration/login/login-hi.spec.js

```diff
@@ -0,0 +1,39 @@
+import { Selector } from "testcafe";
+import { googleAuthUser } from "../../helpers/loginByGoogle";
```

Comment:
otherwise, similar comments as I've added for the corresponding english test file.